pkgs/by-name/wa/warp-plus/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+
```

Comment:
```suggestion
```

Reply:
I use empty lines to separate logical blocks of inputs (and starting with the second one, to sort them alphabetically).